_packaging/tests.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Python package tests"
+excerpt: "
+  More."
```

Comment:
Instead of "More.", should this follow the same format as the previous section, with "This section provides guidance on..."?